draft-demarco-status-attestations.md

```diff
@@ -306,7 +309,7 @@ When the JWT format is used, the JWT MUST contain the parameters defined in the
 
 | JOSE Payload | Description | Reference |
 | --- | --- | --- |
-| **iss** | Wallet identifier. | {{RFC9126}}, {{RFC7519}} |
+| **iss** | Wallet identifier (MUST be equal to JWK thumbprint of the key in `cnf` Claim of Wallet Instance Attestation). | {{RFC9126}}, {{RFC7519}} |
```

Comment:
`MUST be equal to JWK thumbprint of the key in `cnf` Claim of Wallet Instance Attestation`

this is the italian impl profile, not of interest in the IETF OAuth WG, please remove. IT is also not compatible with the DE approach, while we have achieved a compromise that I want to respect in any place.